2022-03-10 13:40:16,681 INFO    MainThread:987410 [wandb_setup.py:_flush():75] Loading settings from /home/udacity/.config/wandb/settings
2022-03-10 13:40:16,682 INFO    MainThread:987410 [wandb_setup.py:_flush():75] Loading settings from /home/udacity/genre_classification/check_data/wandb/settings
2022-03-10 13:40:16,682 INFO    MainThread:987410 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'run_group': 'dev', 'project': 'genre_classification_prod'}
2022-03-10 13:40:16,682 WARNING MainThread:987410 [wandb_setup.py:_flush():75] Could not save program above cwd: /home/udacity/miniconda3/envs/mlflow-acbfa4ee4aa9be3158264a605e4861dd8a78622b/bin/pytest
2022-03-10 13:40:16,682 INFO    MainThread:987410 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': None, 'program': '/home/udacity/miniconda3/envs/mlflow-acbfa4ee4aa9be3158264a605e4861dd8a78622b/bin/pytest'}
2022-03-10 13:40:16,682 INFO    MainThread:987410 [wandb_init.py:_log_setup():405] Logging user logs to /home/udacity/genre_classification/check_data/wandb/run-20220310_134016-8h6hrezh/logs/debug.log
2022-03-10 13:40:16,682 INFO    MainThread:987410 [wandb_init.py:_log_setup():406] Logging internal logs to /home/udacity/genre_classification/check_data/wandb/run-20220310_134016-8h6hrezh/logs/debug-internal.log
2022-03-10 13:40:16,682 INFO    MainThread:987410 [wandb_init.py:init():439] calling init triggers
2022-03-10 13:40:16,682 INFO    MainThread:987410 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-10 13:40:16,682 INFO    MainThread:987410 [wandb_init.py:init():492] starting backend
2022-03-10 13:40:16,682 INFO    MainThread:987410 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-10 13:40:16,686 INFO    MainThread:987410 [backend.py:ensure_launched():219] starting backend process...
2022-03-10 13:40:16,688 INFO    MainThread:987410 [backend.py:ensure_launched():224] started backend process with pid: 987438
2022-03-10 13:40:16,701 INFO    MainThread:987410 [wandb_init.py:init():501] backend started and connected
2022-03-10 13:40:16,702 INFO    MainThread:987410 [wandb_init.py:init():565] updated telemetry
2022-03-10 13:40:16,727 INFO    MainThread:987410 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-10 13:40:17,284 INFO    MainThread:987410 [wandb_run.py:_on_init():1759] communicating current version
2022-03-10 13:40:17,416 INFO    MainThread:987410 [wandb_run.py:_on_init():1763] got version response 
2022-03-10 13:40:17,416 INFO    MainThread:987410 [wandb_init.py:init():625] starting run threads in backend
2022-03-10 13:40:18,024 INFO    MainThread:987410 [wandb_run.py:_console_start():1733] atexit reg
2022-03-10 13:40:18,024 INFO    MainThread:987410 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-03-10 13:40:18,024 INFO    MainThread:987410 [wandb_run.py:_redirect():1611] Redirecting console.
2022-03-10 13:40:18,033 INFO    MainThread:987410 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-10 13:40:18,033 INFO    MainThread:987410 [wandb_init.py:init():664] run started, returning control to user process
2022-03-10 13:40:20,946 INFO    MainThread:987410 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-10 13:40:20,946 INFO    MainThread:987410 [wandb_run.py:_restore():1674] restore
2022-03-10 13:40:23,542 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 1083
}

2022-03-10 13:40:23,656 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 1083
}

2022-03-10 13:40:23,964 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 1083
}

2022-03-10 13:40:24,301 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 4794
}

2022-03-10 13:40:24,402 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 5409
}

2022-03-10 13:40:24,504 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 5409
}

2022-03-10 13:40:24,605 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 5409
}

2022-03-10 13:40:24,706 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 5409
}

2022-03-10 13:40:24,807 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 5409
}

2022-03-10 13:40:24,908 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1083
  total_bytes: 5409
}

2022-03-10 13:40:25,009 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5409
  total_bytes: 5409
}

2022-03-10 13:40:25,110 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5409
  total_bytes: 5409
}

2022-03-10 13:40:25,211 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5409
  total_bytes: 5409
}

2022-03-10 13:40:25,312 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5409
  total_bytes: 5409
}

2022-03-10 13:40:25,413 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5409
  total_bytes: 5409
}

2022-03-10 13:40:25,514 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5409
  total_bytes: 5409
}

2022-03-10 13:40:25,768 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5409
  total_bytes: 5409
}

2022-03-10 13:40:26,269 INFO    MainThread:987410 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5409
  total_bytes: 5409
}
local_info {
}

2022-03-10 13:40:27,372 INFO    MainThread:987410 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-10 13:40:27,372 INFO    MainThread:987410 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-10 13:40:27,373 INFO    MainThread:987410 [wandb_run.py:_footer_sync_info():2822] logging synced files
